### PR TITLE
[Async Clipboard article] Fix iframe permissions policy attributes, add explanation

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Fixes the attributes that had an unfortunate line break.
- Add explanation about permissions policy.

CC: @dway123

### CODE REVIEW

File: src/site/content/en/blog/async-clipboard/index.md [hunk 1]
```diff
@@ -311,6 +306,21 @@ setTimeout(async () => {
 }, 2000);
 ```
 
+## Permissions policy integration
+
+In order to use the API in iframes, you need to enable it by making use of the
```
Comment (jpmedley) [inline src/site/content/en/blog/async-clipboard/index.md:311]:
```suggestion
To use the API in iframes, you need to enable it with
```

File: src/site/content/en/blog/async-clipboard/index.md [hunk 2]
```diff
@@ -311,6 +306,21 @@ setTimeout(async () => {
...
+[Permissions Policy](https://w3c.github.io/webappsec-permissions-policy/),
+which defines a mechanism that allows developers to selectively enable and
+disable various browser features and APIs. Concretely, you need to pass either
```
Comment (jpmedley) [inline src/site/content/en/blog/async-clipboard/index.md:314]:
```suggestion
which defines a mechanism that allows for selectively enabling and
disabling various browser features and APIs. Concretely, you need to pass either
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@tomayac Good move on adding the permission policy info.

**Comment by tomayac** [comment]
Thanks, @jpmedley. PTAL.

**Review by jpmedley — APPROVED** [review]

**Comment by dway123** [comment]
Thank you for this change! :)

**Pull request merged**